### DISCUSSION TITLE
How to update Swar?

### DISCUSSION THREAD

**Discussion by thust777** [swar/Swar-Chia-Plot-Manager#337] [Q&A]
Hi folks,

I'd like to update Swar to the new build. Is it as simple as downloading the zip, unzipping it, and replacing the old contents with the new unzipped contents in, for example, C:\Users\User_Name\Downloads\Swar-Chia-Plot-Manager-main ? 

Thanks!

**Comment by z226** [comment]
I tought stop the stop the swar, replace all files excluding the config and run again) since the chia process run at the background.
Also check if there is any difference or new parameter at the config file.
the question  - Do I need to stop the plots to update?

**Comment by cdeniz** [comment]
I did as you mentioned. But  i got some errors. What is the proper way to upgrade ?

**Comment by z226** [comment]
Excute the command to stop. Restart your pc. Erase all temp folders
Erase the program and copy the new one

**Reply by cdeniz** [reply]
Thank you very much. it works. New parameters in config.yaml file must keep. But I deleted them.  It was my mistake.
